src/webhook/pkg/utils/utils.go

```diff
@@ -58,62 +54,40 @@ func GetResourceRequest(pod *corev1.Pod, resource corev1.ResourceName) int64 {
 	return requestQuantity.Value()
 }
 
-// MergePodResourceRequirements merges enumerated requirements with default requirements
-// it annotates the pod with information about what requirements were modified
-func MergePodResourceRequirements(pod *corev1.Pod, defaultRequirements *corev1.ResourceRequirements) {
-	annotations := []string{}
+// GetResourceLimitQuantity finds and returns the limit quantity for a specific resource.
+func GetResourceLimitQuantity(pod *corev1.Pod, resourceName corev1.ResourceName) resource.Quantity {
+	limitQuantity := resource.Quantity{}
 
-	for i := range pod.Spec.Containers {
-		annotations = mergeContainerResources(&pod.Spec.Containers[i], defaultRequirements, "container", annotations)
+	switch resourceName {
+	case corev1.ResourceCPU:
+		limitQuantity = resource.Quantity{Format: resource.DecimalSI}
+	case corev1.ResourceMemory:
+		limitQuantity = resource.Quantity{Format: resource.BinarySI}
+	default:
+		limitQuantity = resource.Quantity{Format: resource.DecimalSI}
```

Comment:
Optional discussion point. What do you think about refactoring this out to something like `getResourceQuantity(resourceName)` so that it can be shared by `GetResourceLimitQuantity` and `GetResourceRequestQuantity`?

I was trying to think if we could share more of the code between them, but I'm not sure we should go down the reflection route!

Reply:
I can refactor this as well.